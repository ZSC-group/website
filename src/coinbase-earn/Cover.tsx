import * as React from "react"
import HeroMobile from "src/coinbase-earn/hero-mobile.png"
import Hero from "src/coinbase-earn/hero.png"
import { useScreenSize } from "src/layout/ScreenSize"
import { WHEN_MOBILE, flex } from "src/estyles"
import { css } from "@emotion/react"

export default function Cover() {
  const { isMobile } = useScreenSize()
  return (
    <div css={rootCss}>
      <img src={isMobile ? HeroMobile.src : Hero.src} css={imageCss} />
    </div>
  )
}

const rootCss = css(flex, {
  width: "100%",
  backgroundColor: "#1652EB",
  height: 450,
  alignItems: "center",
  [WHEN_MOBILE]: {
    height: 360,
    maxHeight: "50vh",
  },
})

const imageCss = {
  maxWidth: 1050,
  minWidth: 600,
  width: "100%",
  height: "100%",
  [WHEN_MOBILE]: {
    width: "100vw",
    height: "100%",
  },
}
