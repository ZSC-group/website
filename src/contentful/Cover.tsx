import { documentToReactComponents } from "@contentful/rich-text-react-renderer"
import { css } from "@emotion/react"
import {
  flex,
  flexRow,
  fonts,
  WHEN_DESKTOP,
  WHEN_MOBILE,
  WHEN_TABLET,
  WHEN_TABLET_AND_UP,
  whiteText,
} from "src/estyles"
import { GridRow } from "src/layout/Grid2"
import { TABLET_BREAKPOINT } from "src/shared/Styles"
import { CoverContentType } from "src/utils/contentful"
import renderers from "src/contentful/nodes/enodes"
import Button, { SIZE } from "src/shared/Button.3"
import { useScreenSize } from "src/layout/ScreenSize"
import { BLOCKS } from "@contentful/rich-text-types"

const OPTIONS = {
  renderNode: {
    ...renderers,
    [BLOCKS.HEADING_1]: (_, children: string) => {
      return <h2 css={fonts.h1}>{children}</h2>
    },
  },
}

export default function Cover(props: CoverContentType) {
  const { isMobile } = useScreenSize()
  const size = isMobile
    ? props.imageMobile?.fields?.file?.details.image
    : props.imageDesktop?.fields?.file?.details.image
  return (
    <GridRow
      columns={2}
      darkMode={props.darkMode}
      wrapperCss={wrapperCss}
      css={props.illoFirst ? imageFirstRootCss : rootCss}
    >
      <div css={contentCss}>
        <h1
          css={css(
            props.superSize ? titleCss : fonts.h1,
            centerMobileCss,
            props.darkMode && whiteText
          )}
        >
          {props.title}
        </h1>
        <span css={props.darkMode ? subtitleDarkMode : centerMobileCss}>
          {documentToReactComponents(props.subTitle, OPTIONS)}
        </span>
        <div css={linkAreaCss}>
          {props.links?.map((link) => (
            <Button
              align={"center"}
              key={link.sys.id}
              size={isMobile ? SIZE.fullWidth : SIZE.normal}
              kind={link.fields.kind}
              text={link.fields.words}
              href={link.fields.href}
            />
          ))}
        </div>
      </div>
      <div css={illoCss}>
        <picture>
          <source
            media={`(max-width: ${TABLET_BREAKPOINT}px)`}
            srcSet={props.imageMobile?.fields.file.url}
          />
          <img
            width={size?.width}
            height={size?.height}
            css={css(

              props.illoFirst ? imageFirstCss : imageCss,

              props.verticalPosition === "flushBottomText" && flushBottomCss

            )}
            src={props.imageDesktop?.fields.file.url}
            alt={props.imageDesktop?.fields.description}
          />
        </picture>
      </div>
    </GridRow>
  )
}

const wrapperCss = css(flex, {
  alignItems: "center",
  justifyContent: "center",
  [WHEN_MOBILE]: {
    alignContent: "center",
    minHeight: "100vh",
  },
  [WHEN_TABLET]: {
    minHeight: "100vh",
    height: "fit-content",
  },
  [WHEN_DESKTOP]: {
    height: "100vh",
    minHeight: "fit-content",
    maxHeight: "80vw",
  },
})

const rootCss = css({
  gridTemplateAreas: `"content illo"`,
  overflow: "visible",
  [WHEN_MOBILE]: {
    alignContent: "center",
    flexDirection: "column-reverse",
  },
})

const imageFirstRootCss = css(rootCss, {
  gridTemplateAreas: `"illo content"`,
})

const titleCss = css(fonts.h1, {
  [WHEN_DESKTOP]: {
    fontSize: 68,
    lineHeight: "68px",
  },
  [WHEN_MOBILE]: {
    textAlign: "center",
    fontSize: 36,
    lineHeight: "38px",
  },
})

const centerMobileCss = css({
  [WHEN_MOBILE]: {
    textAlign: "center",
  },
})

const subtitleDarkMode = css(whiteText, centerMobileCss, {
  "h1, h2, h3, h4, p": whiteText,
})

const contentCss = css(flex, {
  justifySelf: "center",
  justifyContent: "center",
  flex: 1,
  gridArea: "content",
  [WHEN_TABLET_AND_UP]: {
    paddingTop: 56,
    paddingBottom: 48,
    minWidth: 320,
  },
  [WHEN_MOBILE]: {
    padding: 16,
    maxWidth: 450,
    alignSelf: "center",
  },
})

const linkAreaCss = css(flexRow, {
  [WHEN_MOBILE]: {
    flexDirection: "column",
    "& > div": {
      marginBottom: 24,
    },
  },
  [WHEN_TABLET_AND_UP]: {
    "& > div": {
      marginRight: 24,
      justifyContent: "center",
    },
  },
})

const illoCss = css({
  display: "flex",
  gridArea: "illo",
  position: "relative",
  [WHEN_MOBILE]: {
    width: "100vw",
    justifyContent: "center",
    alignItems: "center",
  },
})

const imageCss = css({
  overflow: "visible",
  [WHEN_MOBILE]: {
    overflow: "inherit",
  },

})

const flushBottomCss = css({
  [WHEN_TABLET_AND_UP]: {
    bottom: 0,
    position: "absolute",
  },
})

const imageFirstCss = css(imageCss, {
  right: 0,
})
