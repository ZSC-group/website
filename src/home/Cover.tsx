/** @jsx jsx */
import {jsx, css, keyframes} from "@emotion/core"
import CoverContent from "src/home/CoverContent"
import { colors } from "src/styles"
import celoAsStarsMobileLong from "src/home/celo-sky-mobile.svg"
import celoAsStarsMobileShort from "src/home/celo-sky-mobile-short.svg"
import celoAsStarsTablet from "src/home/celo-sky-tablet.svg"
import celoAsStarsDesktop from "src/home/celo-sky-desktop.svg"

import examplePhones from "src/home/app-examples@2x.png"
import Stats from "./stats/Stats"
import { flex, WHEN_DESKTOP, WHEN_MOBILE, WHEN_TABLET, WHEN_LONG_PHONE } from "src/estyles"
import { useScreenSize } from "src/layout/ScreenSize"
import { NameSpaces, useTranslation } from "src/i18n"

export default function Cover() {
  const {isDesktop, isTablet, bannerHeight} = useScreenSize()
  const {t} = useTranslation(NameSpaces.home)
  return (
    <div css={css(rootCss, {paddingTop: bannerHeight, [WHEN_MOBILE]: {
      minHeight: `calc(100vh - ${bannerHeight}px)`,
    }})}>
      <div  css={css(backgroundArea, {top: bannerHeight, height: `calc(100% - ${bannerHeight}px)`})} />
      <div css={useableArea}>
        <CoverContent />
        {(isDesktop || isTablet) && <picture>
          {/* add web p, 2x, tablet, and mobile sources*/}
          <img alt={t("coverPhonesImage")} src={examplePhones} width={1016} height={524} css={featureImageCss} />
        </picture>}
      </div>

      {isDesktop && <Stats /> }
  </div>
  )
}

const rootCss = css(flex,{
  position: "relative",
  alignSelf: "center",
  alignItems: "center",
  backgroundColor: colors.dark,
  width: "100%",
  maxWidth: "100vw",
  marginBottom: 500,
  [WHEN_MOBILE]: {
    justifyContent: "center",
  },
  ["@media (max-height: 568px)"]: {
    justifyContent: "flex-end",
    paddingBottom: 30,
  },
  [WHEN_TABLET]: {
    paddingTop: 60,
    width: '100vw',
    height: "100vh",
    marginBottom: 60,
  },
  [WHEN_DESKTOP]: {
    paddingTop: 0,
  }
})

const starKeyFrames =  keyframes`
  from {
    opacity: 0.1;
    transform: scale(1.02);
  }

  25% {
    opacity: 0.5
  }

  to {
    opacity: 1;
    transform: scale(1);
  }
`


const backgroundArea = css({
  position: "absolute",
  backgroundRepeat: "no-repeat",
  backgroundSize: "cover",
  animationIterationCount: 1,
  animationFillMode: "both",
  animationDelay: "50ms",
  animationDuration: "6s",
  animationName: starKeyFrames,
  animationTimingFunction: "ease-in-out",
  opacity: 0.1,
  width: '100%',
  backgroundColor: colors.dark,
  [WHEN_LONG_PHONE]: {
    backgroundImage: `url(${celoAsStarsMobileLong})`,
  },
  [WHEN_MOBILE]: {
    backgroundImage: `url(${celoAsStarsMobileShort})`,
  },
  [WHEN_TABLET]: {
    width: '100vw',
    height: "100vh",
    backgroundImage: `url(${celoAsStarsTablet})`,
  },
  [WHEN_DESKTOP]: {
    width: 1262,
    height: 1067,
    backgroundPositionY: 40,
    backgroundImage: `url(${celoAsStarsDesktop})`,
  }
})

const featureImageCss = css({

})

const useableArea = css(flex,{
  alignItems: "center",
  zIndex: 10,
  [WHEN_DESKTOP]: {
    width: 1262,
    height: 1067 + 50,
    zIndex: 20,
  },
  [WHEN_TABLET]: {
    paddingTop: 72,
  },
  [WHEN_MOBILE]: {
    paddingTop: 24
  }
})