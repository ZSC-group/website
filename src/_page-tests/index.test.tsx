import HomePage from "pages/index"
import * as React from "react"
import * as renderer from "react-test-renderer"
import { TestProvider } from "src/_page-tests/test-utils"
import { ContentfulPage, GridRowContentType } from "src/utils/contentful"
import { Props as BlurbProps } from "src/contentful/grid2-cells/Blurb"
import { Entry } from "contentful"
import { BLOCKS } from "@contentful/rich-text-types"
import { Props as HomeCoverProps } from "src/home/Cover"

function blurbFactory(unique): Entry<BlurbProps> {
  return {
    metadata: { tags: [] },
    sys: {
      id: unique,
      type: "",
      locale: "en",
      createdAt: "2021-07-01",
      updatedAt: "2021-07-01",
      contentType: { sys: { id: "proposition", type: "Link", linkType: "ContentType" } },
    },
    fields: {
      isNaturalSize: false,
      newIcon: false,
      title: `Blurb ${unique}`,
      body: {
        nodeType: BLOCKS.DOCUMENT,
        content: [
          {
            nodeType: BLOCKS.PARAGRAPH,
            content: [{ nodeType: "text", value: "Hello World", marks: [], data: {} }],
            data: {},
          },
        ],
        data: {},
      },
    },
    toPlainObject: () => this,
    update: () => this,
  }
}
// const asset: Asset = {
//   sys: {
//     type: "image",
//     id: "231",
//     createdAt: "",
//     updatedAt: "",
//     locale: "",
//     contentType: {
//       sys: { id: "1", linkType: "ContentType", type: "Link" },
//     },
//   },
//   fields: {
//     title: "Image",
//     description: "Description",
//     file: {
//       url: "/celo.jpg",
//       details: {
//         size: 100,
//         image: {
//           width: 100,
//           height: 100,
//         },
//       },
//       fileName: "celo.jpg",
//       contentType: "img/jpg",
//     },
//   },
//   metadata: { tags: [] },
//   toPlainObject: () => this,
// }

const TestData: ContentfulPage<GridRowContentType | HomeCoverProps> = {
  title: "Celo Home",
  description: "A description of Celo",
  slug: "home",
  darkNav: false,
  sections: [
    {
      toPlainObject: () => this,
      update: () => this,
      metadata: { tags: [] },
      sys: {
        id: "2",
        locale: "en",
        type: "",
        createdAt: "2021-07-01",
        updatedAt: "2021-07-01",
        contentType: { sys: { id: "heading", type: "Link", linkType: "ContentType" } },
      },
      fields: {
        title: "Test",
        subTitle: {
          nodeType: BLOCKS.DOCUMENT,
          content: [
            {
              nodeType: BLOCKS.PARAGRAPH,
              content: [{ nodeType: "text", value: "Built for Us", marks: [], data: {} }],
              data: {},
            },
          ],
          data: {},
        },
        marquee: ["One", "Two"],
        links: [],
        darkMode: true,
      },
    },
    {
      metadata: { tags: [] },
      sys: {
        id: "1",
        type: "",
        locale: "en",
        createdAt: "2021-07-01",
        updatedAt: "2021-07-01",
        contentType: { sys: { id: "grid-row", type: "Link", linkType: "ContentType" } },
      },
      fields: {
        id: "test",
        columns: 3,
        cells: [blurbFactory("romeo"), blurbFactory("soma"), blurbFactory("rico")],
      },
      toPlainObject: () => this,
      update: () => this,
    },
  ],
}

describe("HomePage", () => {
  it("renders", async () => {
    const tree = renderer
      .create(
        <TestProvider>
          <HomePage
            darkNav={false}
            title={TestData.title}
            description={TestData.description}
            slug={TestData.slug}
            sections={TestData.sections}
          />
        </TestProvider>
      )
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
