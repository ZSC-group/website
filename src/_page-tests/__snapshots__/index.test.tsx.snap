// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomePage renders 1`] = `
@keyframes animation-0 {
  0% {
    letter-spacing: -0.5em;
    -webkit-filter: blur(6px);
    filter: blur(6px);
    opacity: 0.10;
  }

  15% {
    -webkit-filter: blur(0px);
    filter: blur(0px);
    opacity: 1;
    letter-spacing: 0em;
  }

  90% {
    -webkit-filter: blur(0px);
    filter: blur(0px);
    opacity: 1;
    letter-spacing: 0em;
  }

  100% {
    letter-spacing: -0.5em;
    -webkit-filter: blur(6px);
    filter: blur(6px);
    opacity: 0.10;
  }
}

.emotion-0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  position: relative;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  overflow: hidden;
  max-width: 100vw;
}

.emotion-1 {
  box-sizing: border-box;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  background-repeat: no-repeat;
  -webkit-background-size: cover;
  background-size: cover;
  box-shadow: inset 0px -50px 37px -25px #111214;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -ms-flex-pack: center;
  -webkit-justify-content: center;
  justify-content: center;
  background-image: url(/celo.jpg);
  box-sizing: border-box;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  overflow: hidden;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-align-self: center;
  -ms-flex-item-align: center;
  align-self: center;
  width: 100%;
  background-color: #111214;
}

@media (max-width: 576px) {
  .emotion-1 {
    -webkit-align-content: center;
    -ms-flex-line-pack: center;
    align-content: center;
    min-height: 90vh;
  }
}

@media (min-width: 576px) and (max-width: 992px) {
  .emotion-1 {
    min-height: 80vh;
    height: -webkit-fit-content;
    height: -moz-fit-content;
    height: fit-content;
  }
}

@media (min-width: 992px) {
  .emotion-1 {
    height: 85vh;
    min-height: -webkit-fit-content;
    min-height: -moz-fit-content;
    min-height: fit-content;
    max-height: 80vw;
  }
}

.emotion-2 {
  box-sizing: border-box;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-align-self: center;
  -ms-flex-item-align: center;
  align-self: center;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  padding-left: 12px;
  padding-right: 12px;
  width: 100%;
  max-width: 100vw;
  overflow: hidden;
  -webkit-box-flex-wrap: wrap;
  -webkit-flex-wrap: wrap;
  -ms-flex-wrap: wrap;
  flex-wrap: wrap;
  -webkit-column-gap: 24px;
  column-gap: 24px;
  grid-template-areas: "content illo";
  overflow: visible;
}

@media (min-width: 576px) and (max-width: 992px) {
  .emotion-2 {
    display: grid;
    grid-auto-rows: auto;
    -webkit-align-self: center;
    -ms-flex-item-align: center;
    align-self: center;
    -webkit-flex-direction: row;
    -ms-flex-direction: row;
    flex-direction: row;
    padding-right: 24px;
    padding-left: 24px;
    width: 100%;
    max-width: 982px;
  }
}

@media (min-width: 992px) {
  .emotion-2 {
    display: grid;
    grid-auto-rows: auto;
    -webkit-align-self: center;
    -ms-flex-item-align: center;
    align-self: center;
    width: 100%;
    max-width: 1104px;
  }
}

@media (min-width: 992px) {
  .emotion-2 {
    grid-template-columns: 1fr 1fr;
  }
}

@media (min-width: 576px) and (max-width: 992px) {
  .emotion-2 {
    grid-template-columns: 1fr 1fr;
  }
}

@media (max-width: 576px) {
  .emotion-2 {
    -webkit-align-content: center;
    -ms-flex-line-pack: center;
    align-content: center;
    -webkit-flex-direction: column;
    -ms-flex-direction: column;
    flex-direction: column;
  }
}

.emotion-3 {
  box-sizing: border-box;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  justify-self: center;
  -webkit-box-pack: center;
  -ms-flex-pack: center;
  -webkit-justify-content: center;
  justify-content: center;
  -webkit-flex: 1;
  -ms-flex: 1;
  flex: 1;
  grid-area: content;
}

@media (min-width: 576px) {
  .emotion-3 {
    padding-top: 56px;
    padding-bottom: 16px;
    min-width: 320px;
  }
}

@media (max-width: 576px) {
  .emotion-3 {
    padding: 16px;
    max-width: 450px;
    -webkit-align-self: center;
    -ms-flex-item-align: center;
    align-self: center;
  }
}

.emotion-4 {
  font-family: EB Garamond,eb-garamond,Garamond,serif;
  font-size: 48px;
  line-height: 56;
  text-rendering: geometricPrecision;
  color: #111214;
  font-weight: normal;
  margin: 0;
  margin-block-start: 0;
  margin-block-end: 0;
  line-height: 56px;
  color: #FFFFFF;
}

@media (max-width: 576px) {
  .emotion-4 {
    font-family: EB Garamond,eb-garamond,Garamond,serif;
    font-size: 36px;
    line-height: 40;
    text-rendering: geometricPrecision;
    color: #111214;
    font-weight: normal;
    margin: 0;
    margin-block-start: 0;
    margin-block-end: 0;
    line-height: 40px;
  }
}

@media (max-width: 576px) {
  .emotion-4 {
    text-align: center;
  }
}

@media (max-width: 576px) {
  .emotion-4 {
    color: #FFFFFF;
  }
}

.emotion-5 {
  -webkit-animation: animation-0;
  animation: animation-0;
  -webkit-animation-timing-function: cubic-bezier(0.250, 0.460, 0.450, 0.940);
  animation-timing-function: cubic-bezier(0.250, 0.460, 0.450, 0.940);
  -webkit-animation-play-state: running;
  animation-play-state: running;
  -webkit-animation-iteration-count: 1;
  animation-iteration-count: 1;
  -webkit-animation-fill-mode: both;
  animation-fill-mode: both;
  -webkit-animation-duration: 3000ms;
  animation-duration: 3000ms;
  padding-left: 12px;
  min-width: 280px;
  display: inline-block;
  text-align: left;
}

@media (max-width: 576px) {
  .emotion-5 {
    min-width: 180px;
  }
}

.emotion-6 {
  color: #FFFFFF;
}

@media (max-width: 576px) {
  .emotion-6 {
    color: #FFFFFF;
  }
}

@media (max-width: 576px) {
  .emotion-6 {
    text-align: center;
  }
}

.emotion-6 h1,
.emotion-6 h2,
.emotion-6 h3,
.emotion-6 h4,
.emotion-6 p {
  color: #FFFFFF;
}

@media (max-width: 576px) {
  .emotion-6 h1,
  .emotion-6 h2,
  .emotion-6 h3,
  .emotion-6 h4,
  .emotion-6 p {
    color: #FFFFFF;
  }
}

.emotion-7 {
  box-sizing: border-box;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  margin-top: 48px;
}

@media (max-width: 576px) {
  .emotion-7 {
    -webkit-flex-direction: column;
    -ms-flex-direction: column;
    flex-direction: column;
  }

  .emotion-7>div {
    margin-bottom: 24px;
  }
}

@media (min-width: 576px) {
  .emotion-7>div {
    margin-right: 24px;
    -webkit-box-pack: center;
    -ms-flex-pack: center;
    -webkit-justify-content: center;
    justify-content: center;
  }
}

.emotion-8 {
  grid-column: span 2;
  justify-self: center;
}

.emotion-9 {
  box-sizing: border-box;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  opacity: 0;
  transition-property: opacity;
  transition-duration: 550ms;
  position: relative;
  top: 200px;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  background-color: #111214;
  border-radius: 16px;
  max-width: 724px;
  padding: 24px 40px;
  z-index: 20;
  box-shadow: 0px 1px 16px rgba(255, 255, 255, 0.16),0px 0px 24px rgba(78, 236, 255, 0.6);
}

@media (min-width: 576px) and (max-width: 992px) {
  .emotion-9 {
    top: 170px;
  }
}

@media (max-width: 576px) {
  .emotion-9 {
    max-width: 256px;
    margin: 16px 60px;
    -webkit-flex-basis: 0%;
    -ms-flex-preferred-size: 0%;
    flex-basis: 0%;
    padding: 24px 0px;
    -webkit-filter: drop-shadow(0px 3px 16px rgba(78, 236, 255, 0.7)) drop-shadow(0px 2px 54px #353D45);
    filter: drop-shadow(0px 3px 16px rgba(78, 236, 255, 0.7)) drop-shadow(0px 2px 54px #353D45);
    box-shadow: 0px 2px 54px #353D45;
    border-radius: 4px;
    top: 0;
  }
}

.emotion-10 {
  font-family: Jost,futura-pt,futura,sans-serif;
  margin-top: 8px;
  font-weight: 500;
  font-size: 12px;
  line-height: 16px;
  letter-spacing: 3px;
  text-transform: uppercase;
  font-size: 20px;
  letter-spacing: 6px;
  margin-top: 4px;
  padding-bottom: 10px;
}

@media (max-width: 576px) {
  .emotion-10 {
    font-size: 10px;
    margin-bottom: 4px;
    padding: 0;
    text-align: center;
  }
}

.emotion-11 {
  color: #FFFFFF;
  -webkit-text-decoration: none;
  text-decoration: none;
}

.emotion-12 {
  box-sizing: border-box;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  justify-content: space-between;
  margin: 8px 0px;
  max-width: 644px;
}

@media (max-width: 576px) {
  .emotion-12 {
    -webkit-flex-direction: column;
    -ms-flex-direction: column;
    flex-direction: column;
    margin: 16px 0px;
  }
}

.emotion-13 {
  box-sizing: border-box;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -ms-flex-pack: center;
  -webkit-justify-content: center;
  justify-content: center;
}

@media (max-width: 576px) {
  .emotion-13 {
    -webkit-flex-direction: column;
    -ms-flex-direction: column;
    flex-direction: column;
  }
}

.emotion-14 {
  font-family: EB Garamond,eb-garamond,Garamond,serif;
  color: #FFFFFF;
  font-size: 24px;
  line-height: 1.2;
  text-align: center;
  margin-right: 10px;
}

@media (max-width: 576px) {
  .emotion-14 {
    font-size: 20px;
    margin-right: 0;
  }
}

.emotion-15 {
  font-family: Jost,futura-pt,futura,sans-serif;
  color: #EDEEEF;
  font-size: 16px;
  line-height: 34px;
  text-align: center;
  margin-right: 40px;
  white-space: nowrap;
  text-decoration-line: none;
}

@media (max-width: 576px) {
  .emotion-15 {
    font-size: 12px;
    margin-right: 0;
  }
}

.emotion-21 {
  font-family: Jost,futura-pt,futura,sans-serif;
  color: #EDEEEF;
  font-size: 16px;
  line-height: 34px;
  text-align: center;
  margin-right: 40px;
  white-space: nowrap;
  text-decoration-line: none;
}

@media (max-width: 576px) {
  .emotion-21 {
    font-size: 12px;
    margin-right: 0;
  }
}

.emotion-21:hover {
  text-decoration-line: underline;
}

.emotion-22 {
  box-sizing: border-box;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  overflow: hidden;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-align-self: center;
  -ms-flex-item-align: center;
  align-self: center;
  width: 100%;
}

.emotion-23 {
  box-sizing: border-box;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-align-self: center;
  -ms-flex-item-align: center;
  align-self: center;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  padding-left: 12px;
  padding-right: 12px;
  width: 100%;
  max-width: 100vw;
  overflow: hidden;
  -webkit-box-flex-wrap: wrap;
  -webkit-flex-wrap: wrap;
  -ms-flex-wrap: wrap;
  flex-wrap: wrap;
  -webkit-column-gap: 24px;
  column-gap: 24px;
}

@media (min-width: 576px) and (max-width: 992px) {
  .emotion-23 {
    display: grid;
    grid-auto-rows: auto;
    -webkit-align-self: center;
    -ms-flex-item-align: center;
    align-self: center;
    -webkit-flex-direction: row;
    -ms-flex-direction: row;
    flex-direction: row;
    padding-right: 24px;
    padding-left: 24px;
    width: 100%;
    max-width: 982px;
  }
}

@media (min-width: 992px) {
  .emotion-23 {
    display: grid;
    grid-auto-rows: auto;
    -webkit-align-self: center;
    -ms-flex-item-align: center;
    align-self: center;
    width: 100%;
    max-width: 1104px;
  }
}

@media (min-width: 992px) {
  .emotion-23 {
    grid-template-columns: 1fr 1fr 1fr;
  }
}

@media (min-width: 576px) and (max-width: 992px) {
  .emotion-23 {
    grid-template-columns: 1fr 1fr 1fr;
  }
}

.emotion-24 {
  box-sizing: border-box;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-flex: 1;
  -ms-flex: 1;
  flex: 1;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  justify-content: space-between;
  margin-bottom: 36px;
}

@media (max-width: 576px) {
  .emotion-24 {
    -webkit-align-items: center;
    -webkit-box-align: center;
    -ms-flex-align: center;
    align-items: center;
    -webkit-align-content: center;
    -ms-flex-line-pack: center;
    align-content: center;
  }
}

.emotion-25 {
  box-sizing: border-box;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-flex: 1;
  -ms-flex: 1;
  flex: 1;
}

.emotion-25 p:first-of-type {
  margin-top: 0;
}

.emotion-25 p:last-of-type {
  margin-bottom: 0;
}

@media (max-width: 576px) {
  .emotion-25 {
    -webkit-align-items: center;
    -webkit-box-align: center;
    -ms-flex-align: center;
    align-items: center;
    -webkit-align-content: center;
    -ms-flex-line-pack: center;
    align-content: center;
    text-align: center;
    max-width: 288px;
  }

  .emotion-25 li {
    list-style: none;
  }

  .emotion-25 ul {
    margin-block-start: 0;
    margin-block-end: 0;
    -webkit-margin-start: 0;
    margin-inline-start: 0;
    margin-inlin-eend: 0;
    margin-inlin-eend: 0;
    -webkit-padding-start: 0;
    padding-inline-start: 0;
  }
}

.emotion-26 {
  font-weight: normal;
  margin: 0;
  margin-block-start: 0;
  margin-block-end: 0;
  font-family: Jost,futura-pt,futura,sans-serif;
  font-size: 24px;
  line-height: 28;
  font-weight: 400;
  text-rendering: geometricPrecision;
  color: #111214;
  line-height: 28px;
  margin-top: 16px;
  margin-bottom: 12px;
}

.emotion-27 {
  font-family: EB Garamond,eb-garamond,Garamond,serif;
  font-size: 20px;
  line-height: 28;
  text-rendering: geometricPrecision;
  color: #111214;
  line-height: 28px;
}

<div
  className="emotion-0"
>
  <section
    className="emotion-1"
  >
    <div
      className="emotion-2"
    >
      <div
        className="emotion-3"
      >
        <h1
          className="emotion-4"
        >
          Test
           
          <em
            className="emotion-5"
          >
            One
          </em>
        </h1>
        <span
          className="emotion-6"
        >
          <p>
            Built for Us
          </p>
        </span>
        <div
          className="emotion-7"
        />
      </div>
      <div
        className="emotion-8"
      >
        <figure
          aria-hidden={true}
          className="emotion-9"
        >
          <figcaption
            className="emotion-10"
          >
            <a
              className="emotion-11"
              href="https://explorer.celo.org"
              rel="noreferrer"
              target="_blank"
            >
              PLATFORM STATS
            </a>
          </figcaption>
          <div
            className="emotion-12"
          >
            <div
              className="emotion-13"
            >
              <span
                aria-labelledby="stat-tx"
                className="emotion-14"
              />
              <span
                className="emotion-15"
                id="stat-tx"
              >
                Total transactions
              </span>
            </div>
            <div
              className="emotion-13"
            >
              <span
                aria-labelledby="stat-time"
                className="emotion-14"
              >
                0s
              </span>
              <span
                className="emotion-15"
                id="stat-time"
              >
                Average block time
              </span>
            </div>
            <div
              className="emotion-13"
            >
              <span
                aria-labelledby="carbon"
                className="emotion-14"
              >
                641
              </span>
              <a
                className="emotion-21"
                href="https://www.wren.co/profile/celo"
                id="carbon"
                rel="noopener"
                target="_blank"
              >
                Days carbon negative
              </a>
            </div>
          </div>
        </figure>
      </div>
    </div>
  </section>
  <section
    className="emotion-22"
  >
    <div
      className="emotion-23"
      id="test"
    >
      <div
        className="emotion-24"
      >
        <div
          className="emotion-25"
        >
          <h4
            className="emotion-26"
          >
            Blurb romeo
          </h4>
          <p
            className="emotion-27"
          >
            Hello World
          </p>
        </div>
      </div>
      <div
        className="emotion-24"
      >
        <div
          className="emotion-25"
        >
          <h4
            className="emotion-26"
          >
            Blurb soma
          </h4>
          <p
            className="emotion-27"
          >
            Hello World
          </p>
        </div>
      </div>
      <div
        className="emotion-24"
      >
        <div
          className="emotion-25"
        >
          <h4
            className="emotion-26"
          >
            Blurb rico
          </h4>
          <p
            className="emotion-27"
          >
            Hello World
          </p>
        </div>
      </div>
    </div>
  </section>
</div>
`;
